### PR TITLE
Add `delimiters' and related predicates for `RegexpNode`

### PR DESCRIPTION
**Pull request by owst**
As suggested by @bbatsov here:
  https://github.com/rubocop-hq/rubocop/pull/8138#discussion_r439268175

### CODE REVIEW

File: lib/rubocop/ast/node/regexp_node.rb [hunk 1]
```diff
@@ -32,6 +32,11 @@ def content
         children.select(&:str_type?).map(&:str_content).join
       end
 
+      # @return [Bool] if char is one of the delimiters
+      def delimiter?(char)
```
Comment (bbatsov) [inline lib/rubocop/ast/node/regexp_node.rb:36]: I guess you can also add a `delimiters` method that returns the two of them and implement this one in terms of it.

Comment (owst) [reply lib/rubocop/ast/node/regexp_node.rb:51]: Good idea - I took a quick look for related predicates used by RuboCop, so added those too, let me know what you think

**Comment by bbatsov** [comment]
Great work!

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,7 @@
 * [#20](https://github.com/rubocop-hq/rubocop-ast/pull/20): Add option predicates for `RegexpNode`. ([@owst][])
 * [#11](https://github.com/rubocop-hq/rubocop-ast/issues/11): Add `argument_type?` method to make it easy to recognize argument nodes. ([@tejasbubane][])
 * [#31](https://github.com/rubocop-hq/rubocop-ast/pull/31): Use `param === node` to match params, which allows Regexp, Proc, Set, etc. ([@marcandre][])
+* [#41](https://github.com/rubocop-hq/rubocop-ast/pull/41): Add `delimiters' and related predicates for `RegexpNode`. ([@owst][])
```
Comment (bbatsov) [inline CHANGELOG.md:11]: You've made a small mistake with the backticks here.

**Comment by bbatsov** [comment]
Thanks!

**Pull request merged**